spec/views/posts/index.html.erb_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe 'posts/index.html.erb', type: :view do
```

Comment:
- [ ] Spec file should include tests that ensure the correct view template is rendered, as the previous reviewer mentioned kindly consider including those tests
e.g  `expect(response).to render_template(:show)`